.github/workflows/release.yml

```diff
@@ -16,6 +16,9 @@ on:
         required: true
         type: string
 
+permissions:
```

Comment:
I think you can give restrict this permission to the last step only.

Reply:
Ok I thought permissions were action scoped, I will check